pkg/kubelet/eviction/helpers.go

```diff
@@ -169,18 +170,47 @@ func parseThresholdStatement(statement string) (Threshold, error) {
 		return Threshold{}, fmt.Errorf(unsupportedEvictionSignal, signal)
 	}
 
-	quantity, err := resource.ParseQuantity(parts[1])
-	if err != nil {
-		return Threshold{}, err
+	quantityValue := parts[1]
+	if strings.HasSuffix(quantityValue, "%") {
+		percentage, err := parsePercentage(quantityValue)
+		if err != nil {
+			return Threshold{}, err
+		}
+		if percentage < 0 {
```

Comment:
shouldn't this be <= 0?

basically, a percentage must have a positive value?

is there a case where total exhaustion is a good thing?

maybe this also applies to our current literal value support.